docs/userGuide/syntax/icons.md

```diff
@@ -55,6 +55,12 @@ MarkBind currently supports Version 6 of Font Awesome (Free plan). For detailed
 1. You may also append `~class-name` to the end of the octicon name to add `class="class-name"` property to your Octicon (e.g. `:octicon-git-pull-request~icon-large-red:` will generate an Octicon of class *icon-large-red*). You may then add corresponding CSS to `{root}/_markbind/layouts/{layout-name}/styles.css` to customize the style of your Octicon.
 1. If your background is dark, you may use `:octiconlight-*:` to render the icon as white. 
 
+###### Using Bootstrap icons
```

Comment:
1. Lines 3-5 : can you also update in this PR? Version of bootstrap relied on + Maybe delete line 5, which seems too repetitive. 
2. I never noticed before how repetitive these instructions are. I wonder if there is a better way to streamline the instructions? Hmm. Maybe a tooltip or something to see some different example icons would be cool.